src/server/graphql/mutations/login.js

```diff
@@ -66,14 +66,14 @@ const login = {
 
     let userInfo;
     try {
-      userInfo = await auth0Client.tokens.getInfo(auth0Token);
+      const auth0ManagementClient = await auth0MgmtClientBuilder();
+      userInfo = await auth0ManagementClient.getUser({
```

Comment:
As you may have guessed @mattkrick, this was the original site of our woes. It wasn't until I stepped through this section in the debugger that I saw the old auth0Client wasn't returning an exception but a _very_ unhelpful JSON response that would make its way into the `newUser` object below. The actual line of failure was `ensureDate(userInfo.createdAt)`, that check would fail and toss an exception.

Reply:
yeah, instead of throwing an error, they return a string instead of an object. wtf....